gtsam/sfm/ShonanAveraging.h

```diff
@@ -47,12 +47,16 @@ struct GTSAM_EXPORT ShonanAveragingParameters {
   using Anchor = std::pair<size_t, Rot>;
 
   // Paremeters themselves:
```

Comment:
Paremeters -> Parameters

also: following GaussNewtonParams, LevenbergMarquardtParams, etc, shouldn't we call this class "ShonanAveragingParams"?